plugins/outputs/warp10/warp10.go

```diff
@@ -0,0 +1,166 @@
+package warp10
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+type Warp10 struct {
+	Prefix string
+
+	WarpUrl string
+
+	Token string
+}
+
+var sampleConfig = `
+  # prefix for metrics class Name
+  prefix = "telegraf."
+  ## POST HTTP Mode ##
+  # Url name of the Warp 10 server
+  warp_url = "localhost:4242/"
```

Comment:
rather than calling out "POST HTTP MODE", I would say to make it more clear by requiring that users provide a scheme with the url. This way other schemes could be supported in the future as well:

Also do not require that the trailing slash is there:

```
  # Url name of the Warp 10 server
  warp_url = "http://localhost:4242"
```